org.moreunit.swtbot.test/src/org/moreunit/refactoring/RenameMethodTest.java

```diff
@@ -4,6 +4,9 @@
 
 import org.eclipse.jdt.core.IMethod;
 import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.internal.ui.refactoring.RefactoringMessages;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.ltk.internal.ui.refactoring.RefactoringUIMessages;
```

Comment:
I don't know the project's policy on using internals from other bundles. My preference however is to avoid them if at all possible